assets/js/components/notifications/SetupSuccessBannerNotification.js

```diff
@@ -200,6 +200,21 @@ function SetupSuccessBannerNotification() {
 				};
 			}
 
+			if ( 'thank-with-google' === slug ) {
+				winData.description = __(
+					'Thank with Google is visible to your visitors. To see metrics, ',
```

Comment:
This might seem overly pedantic, but we don't need the trailing space on this string...

For such a trivial amendment, I'll apply the change myself to save the round trip.

```suggestion
					'Thank with Google is visible to your visitors. To see metrics,',
```